--- cmake/modules/FindXscreensaver.cmake.orig	2009-02-26 15:18:09.000000000 +0100
+++ cmake/modules/FindXscreensaver.cmake	2009-02-27 17:07:30.704815247 +0100
@@ -27,7 +27,7 @@
   FIND_PATH(XSCREENSAVER_DIR deco ${xscreensaver_alldirs})
 
   set(XSCREENSAVER_CONFIG_DIR)
-  FIND_PATH(XSCREENSAVER_CONFIG_DECO config/deco.xml
+  FIND_PATH(XSCREENSAVER_CONFIG_DECO deco.xml
     ${KDE4_INCLUDE_DIR}
     /usr/
     /usr/local/
